megamek/src/megamek/common/Compute.java

```diff
@@ -6678,7 +6677,7 @@ public static boolean inDeadZone(IGame game, Entity ae, Targetable target) {
                 if ((en.getOwner().equals(other.getOwner()) || (en.getOwner()
                                                                   .getTeam() == other.getOwner().getTeam()))
                     && !en.equals(other)
-                    && (other instanceof SmallCraft)
+                    && ((other instanceof SmallCraft) || other.getTowing() != Entity.NONE || other.getTowedBy() != Entity.NONE)
```

Comment:
Small craft can tow entities?

Reply:
This is code for loading infantry aboard multi-hex units, and yes, trains handle it like Dropships